packages/components/icons/src/WorkflowsIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  IconVariant,
+  generateComponentWithVariants,
+} from '@contentful/f36-icon';
+import { generateIconComponent } from '@contentful/f36-icon';
```

Comment:
Could be consolidated

```suggestion
import {
  IconVariant,
  generateIconComponent,
  generateComponentWithVariants,
} from '@contentful/f36-icon';
```